Sources/TuistGraph/Models/LaunchStyle.swift

```diff
@@ -1,6 +1,16 @@
 import Foundation
+import ProjectDescription
 
 public enum LaunchStyle: Codable {
     case automatically
     case waitForExecutableToBeLaunched
+
+    public static func from(launchStyle: ProjectDescription.LaunchStyle) -> TuistGraph.LaunchStyle {
+        switch launchStyle {
+        case .automatically:
+            return .automatically
+        case .waitForExecutableToBeLaunched:
+            return .waitForExecutableToBeLaunched
+        }
+    }
```

Comment:
I would move this to `Sources/TuistLoader/Models+ManifestMappers/RunAction+ManifestMapper.swift` directly.
`TuistGraph` shouldn't depend on `ProjectDescription`

Reply:
I tried placing the function that maps a ProjectDescription.LaunchStyle to TuistGraph.LaunchStyle in `TuistLoader/Models.ManifestMappers/LaunchStyle+ManifestMapper.swift`, is that okay?